protostar/git/git_repository.py

```diff
@@ -17,6 +17,11 @@ class Submodule:
     path: Optional[Path] = None
 
 
+@dataclass
+class GitStatusResult:
+    staged_file_paths: list[Path]
```

Comment:
This is not used anywhere but in tests, we don't need that value

Reply:
It's used in `install_package_from_repo.py`.

I could've removed this dataclass in favor of a more specific function, however I chose to keep the interface of `GitRepository` similar to the `git cli`.